src/Tokenizer/TokensAnalyzer.php

```diff
@@ -299,6 +299,22 @@ public function isLambda(int $index): bool
         return $startParenthesisToken->equals('(');
     }
 
+    /**
+     * Check if the function under given index is a short arrow lambda.
+     */
```

Comment:
I wanted to achieve a more specific token check here - I'm not sure if this was the best approach.